types/probe-image-size/tslint.json

```
@@ -1 +1,6 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false
+    }
```

```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
this should be updated on really demanding use cases. Inline comment should work